src/java.base/share/classes/java/lang/invoke/VarHandle.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2014, 2020, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2014, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Unless this is backport of OpenJDK changes the Oracle copyright shouldn't change. Pls add an IBM copyright statement.